builq_bench_test.go

```diff
@@ -0,0 +1,73 @@
+package builq_test
+
+import (
+	"testing"
+
+	"github.com/cristalhq/builq"
+)
+
+func BenchmarkBuildNaive(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		var bb builq.Builder
+		bb.Addf("SELECT %s FROM %s", "foo,bar", "table")
+		bb.Addf("WHERE id = %$", 123)
+		query, args, err := bb.Build()
+
+		switch {
```

Comment:
Looks like a test to me, do we need it in benchmarks? If so, could be a testing helper due to all switches being exactly the same

Reply:
Yes, it's needed, price is so low that it doesn't make sense to value it.